code/datums/status_effects/gas.dm

```diff
@@ -5,31 +5,43 @@
 	alert_type = /obj/screen/alert/status_effect/freon
 	var/icon/cube
 	var/can_melt = TRUE
+	var/active
```

Comment:
This is used to check if the status effect is still active after the `do_after` later, so that we don't try and continue a proc that has ceased to exist if the status effect terminates while you are breaking the ice.